assets/tuned/06-cm-tuned-profiles.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tuned-profiles
+  namespace: openshift-cluster-node-tuning
+data:
+  tuned-profiles-data: |
+    openshift: |
+      [main]
+      summary=Optimize systems running OpenShift (parent profile)
+      include=${f:virt_check:virtual-guest:throughput-performance}
+
+      [selinux]
+      avc_cache_threshold=65536
+
+      [net]
+      nf_conntrack_hashsize=131072
+
+      [sysctl]
+      net.ipv4.ip_forward=1
+      kernel.pid_max=>131072
+      net.netfilter.nf_conntrack_max=1048576
+      fs.inotify.max_user_watches=65536
+      net.ipv4.neigh.default.gc_thresh1=8192
+      net.ipv4.neigh.default.gc_thresh2=32768
+      net.ipv4.neigh.default.gc_thresh3=65536
+      net.ipv6.neigh.default.gc_thresh1=8192
+      net.ipv6.neigh.default.gc_thresh2=32768
+      net.ipv6.neigh.default.gc_thresh3=65536
+    openshift-control-plane: |
+      [main]
+      summary=Optimize systems running OpenShift control plane
+      include=openshift
+
+      [sysctl]
+      # ktune sysctl settings, maximizing i/o throughput
+      #
+      # Minimal preemption granularity for CPU-bound tasks:
+      # (default: 1 msec#  (1 + ilog(ncpus)), units: nanoseconds)
+      kernel.sched_min_granularity_ns=10000000
+      # The total time the scheduler will consider a migrated process
+      # "cache hot" and thus less likely to be re-migrated
+      # (system default is 500000, i.e. 0.5 ms)
+      kernel.sched_migration_cost_ns=5000000
+      # SCHED_OTHER wake-up granularity.
+      #
+      # Preemption granularity when tasks wake up.  Lower the value to
+      # improve wake-up latency and throughput for latency critical tasks.
+      kernel.sched_wakeup_granularity_ns=4000000
+    openshift-node: |
+      [main]
+      summary=Optimize systems running OpenShift nodes
+      include=openshift
+
+      [sysctl]
+      net.ipv4.tcp_fastopen=3
+      fs.inotify.max_user_watches=65000
```

Comment:
hmm, this exists in the parent profile as well

Reply:
Right, so where do we set this and what should be the default value? BZ1559061 suggested fs.inotify.max_user_watches=524288 for the node profile.  PR https://github.com/openshift/openshift-ansible/pull/9021 just made it user-configurable.

Reply:
Apparently this is leak is fixed in 1.13